.all-contributorsrc

```diff
@@ -78,6 +78,124 @@
         "code",
         "ideas"
       ]
+    },
+    {
+      "login": "Ndpnt",
+      "name": "Nicolas Dupont",
+      "avatar_url": "https://avatars.githubusercontent.com/u/1098708?v=4",
+      "profile": "https://github.com/Ndpnt",
+      "contributions": [
+        "code",
+        "infra",
+        "doc"
+      ]
+    },
+    {
+      "login": "MattiSG",
+      "name": "Matti Schneider",
+      "avatar_url": "https://avatars.githubusercontent.com/u/222463?v=4",
+      "profile": "https://mattischneider.fr/",
+      "contributions": [
+        "code",
+        "infra",
+        "doc"
+      ]
+    },
+    {
+      "login": "martinratinaud",
+      "name": "Martin Ratinaud. Freelancer.",
+      "avatar_url": "https://avatars.githubusercontent.com/u/4191809?v=4",
+      "profile": "http://bit.ly/work-with-martin",
+      "contributions": [
+        "code",
+        "infra",
+        "doc"
+      ]
+    },
+    {
+      "login": "clementbiron",
+      "name": "Clément Biron",
+      "avatar_url": "https://avatars.githubusercontent.com/u/364319?v=4",
+      "profile": "https://clementbiron.com/",
+      "contributions": [
+        "code",
+        "doc"
+      ]
+    },
+    {
+      "login": "AdrienFines",
+      "name": "AdrienFines",
+      "avatar_url": "https://avatars.githubusercontent.com/u/41912392?v=4",
+      "profile": "https://github.com/AdrienFines",
+      "contributions": [
+        "code",
+        "data"
+      ]
+    },
+    {
+      "login": "vviers",
+      "name": "Vincent Viers",
+      "avatar_url": "https://avatars.githubusercontent.com/u/30295971?v=4",
+      "profile": "https://beta.gouv.fr/",
+      "contributions": [
+        "code",
+        "data",
+        "doc"
+      ]
+    },
+    {
+      "login": "AaronjSugarman",
```

Comment:
No, this contributor contributed only data that should be in contrib, not the engine.
Please refer to the [allocation spreadsheet](https://docs.getgrist.com/rvbxgUiHTW7Q/Contributors).

Reply:
I used the repository contributors as a source of data.
I've just updated using the allocation spreadsheet.